ui/select.reel/select.js

```diff
@@ -254,14 +254,27 @@ var Select = exports.Select = Component.specialize({
         value: function(distance) {
             var currentIndex = this.optionsOverlayComponent.templateObjects.options.iterations.indexOf(this._highlightedOption),
                 newIndex = currentIndex + distance,
-                contentLength = this.optionsOverlayComponent.templateObjects.options.iterations.length;
+                contentLength = this.optionsOverlayComponent.templateObjects.options.iterations.length,
+                optionsHeight = this.optionsOverlayComponent.scrollView.scrollTop + this.optionsOverlayComponent.scrollView.element.offsetHeight;
 
             if (newIndex < -1) {
                 newIndex = contentLength -1;
             }
             if (newIndex != -1 && newIndex != contentLength) {
                 this._highlightedOption = this.optionsOverlayComponent.templateObjects.options.iterations[newIndex % contentLength];
+
+                // scroll to element
+                var highlightedElement = this._highlightedOption.firstElement,
```

Comment:
That's only work if all options have the same height, it's not an issue in our case?